### ISSUE TITLE
Feedback Features section

### ISSUE THREAD

**Issue by cdeil** [gammapy/gammaskynet-gamma2016#20]
- [x] For "map view" I'd again mention Aladin Lite, since all of the features are implemented there and we just use them.
- [x] The search functionality is powered by http://cds.u-strasbg.fr/cgi-bin/Sesame . I suggest to mention that somewhere, either here in the features section or where you first talk about it.
- [x] Catalog view - suggest to remove the footnote. It says that these features will be part of all catalogs, but that's not true. SNRcat doesn't have most of those things. The Fermi catalogs are mostly point-source catalogs, i.e. no extension info. The TeV catalog doesn't have a lightcurve for most sources. Maybe you could just rephrase it to something like this? "The catalog view allows searching and selecting a source from the catalogs in Table 2. For a given source, the information available in that catalog is shown, including .... (list the things you have now).

**Comment by vorugantia** [comment]
See fixes [here](https://github.com/gammapy/gammaskynet-gamma2016/commit/0f5fce3b400255cb05040566085487381c7f1de6).

I just mentioned Sesame here, but I didn't explain it until the Data section so that this section remains as a quick bullet point list. (But I did this in a different commit.) I also made it clear that the map interface is from Aladin Lite.
